src/components/HeaderBar.vue

```diff
@@ -48,12 +48,17 @@ export default Vue.extend({
 })
 </script>
 
-<style lang="scss" scoped>
```

Comment:
Instead of losing the `scoped`, can you use `::v-deep`?

Reply:
Yep, ::v-deep achieves the same.